### PR TITLE
multi-arch-pipeline: start building other artifacts for aarch64

### PR DESCRIPTION
**Pull request by dustymabe**
This adds the metal artifacts as well as the openstack image.

### CODE REVIEW

**Comment by dustymabe** [comment]
There are better ways to do this. We'll start with this for now.

File: jobs/multi-arch-pipeline.Jenkinsfile [hunk 1]
```diff
@@ -285,13 +285,20 @@ stages:
...
+  require_artifacts: [ostree]
   post_commands:
+    - cosa buildextend-qemu
     - cosa kola run --basic-qemu-scenarios
+    - cosa kola run --parallel 4
+    - cosa buildextend-metal
+    - cosa buildextend-metal4k
+    - cosa buildextend-live
+    - kola testiso -S --output-dir tmp/kola-metal
+    - kola testiso -SP --qemu-native-4k --scenarios iso-install --output-dir tmp/kola-metal4k
```
Comment (jlebon) [inline jobs/multi-arch-pipeline.Jenkinsfile:299]: Are the output dirs here copied off the worker so we can archive them? Feels like we're also missing the `archiveArtifacts` step here.

Comment (dustymabe) [reply jobs/multi-arch-pipeline.Jenkinsfile:299]: agree, yes they are copied back. I've got a few more followups to make (kola aws uploads and testing) and some cycles to run in my dev jenkins. Can I add that step then?

Comment (dustymabe) [reply jobs/multi-arch-pipeline.Jenkinsfile:299]: actually there is some more I need to do to get them to get copied back - but yeah will do that in a followup

File: jobs/multi-arch-pipeline.Jenkinsfile [hunk 2]
```diff
@@ -285,13 +285,20 @@ stages:
...
   description: Stage 1 execution base
-  build_artifacts: [qemu]
-  require_artifacts: [ostree]
   prep_commands:
     - cat /cosa/coreos-assembler-git.json
+  require_artifacts: [ostree]
   post_commands:
+    - cosa buildextend-qemu
     - cosa kola run --basic-qemu-scenarios
+    - cosa kola run --parallel 4
```
Comment (dustymabe) [inline jobs/multi-arch-pipeline.Jenkinsfile:294]: Do these two write/overwrite the same directory? Do I need to specify `--output-dir` here too?

Comment (dustymabe) [reply jobs/multi-arch-pipeline.Jenkinsfile]: updated it so that they all specify `--output-dir`

**Review by jlebon — APPROVED** [review]

**Pull request merged**